pkg/restapi/vc/operation/operations.go

```diff
@@ -166,13 +166,14 @@ func New(config *Config) (*Operation, error) {
 		return nil, fmt.Errorf("failed to instantiate new csl status: %w", err)
 	}
 
-	jweEncrypter, jweDecrypter, err := cryptosetup.PrepareJWECrypto(config.KeyManager, config.StoreProvider)
+	jweEncrypter, jweDecrypter, err := cryptosetup.PrepareJWECrypto(config.KeyManager, config.StoreProvider,
+		jose.A256GCM, kms.ECDHES256AES256GCMType)
```

Comment:
I also noticed that I was using `kms.ECDHES256AES256GCM` before instead of `kms.ECDHES256AES256GCMType`, so I fixed that (although I suppose they work the same anyway, I think you intended for us to use `kms.ECDHES256AES256GCMType` @Baha-sk?

Reply:
FooType vs Foo is simply a custom type for code clarity, so yes :+1: